packages/cache/src/internal/cacheUtils.ts

```diff
@@ -123,3 +123,10 @@ export function assertDefined<T>(name: string, value?: T): T {
 
   return value
 }
+
+export function isGhes(): boolean {
```

Comment:
Please update cache actions and other setup-* actions to use this utils instead of their own copies. Not a blocking comment but good to take up as a follow up.

Reply:
Removed this method.